AccountTests/LiveAccountTest.swift

```diff
@@ -54,10 +54,13 @@ open class LiveAccountTest: XCTestCase {
             let passwordUT8 = password.utf8EncodedData
             let stretchedPW = FxAClient10.quickStretchPW(emailUTF8, password: passwordUT8)
             completion(emailUTF8, stretchedPW)
+            expectation.fulfill()
         } else {
             // This is the standard case: signedInUser.json is {}.
             NSLog("Skipping test because signedInUser.json does not include email address.")
-        }
+            expectation.fulfill()
+            self.waitForExpectations(timeout: 10, handler: nil)
```

Comment:
Why is this `wait` here?

Reply:
From https://github.com/mozilla-mobile/firefox-ios/pull/3071#discussion_r135601692, this appeared to let the expectations complete and pass tests. Not sure if this was the correct way to clean up these.

Reply:
You shouldn't need to add this.